common/services/whats-new-content.js

```diff
@@ -45,6 +45,10 @@ const service = {
   fetch: async () => {
     const entries = await service.client.getEntries()
 
+    if (!entries.items && entries.sys.id === 'ServerError') {
+      return { error: 'ServerError', errorMessage: entries.message }
+    }
+
```

Comment:
Do we know for sure this is what happens if there's an error from the client?

I'm just wandering [because the documentation suggests that it raises an exception](https://github.com/contentful/contentful.js#your-first-request):

<img width="318" alt="Screenshot 2022-03-23 at 07 31 31" src="https://user-images.githubusercontent.com/510498/159646523-887b3254-b41f-4ae8-b1b2-d3b5cbd04ea0.png">

Reply:
From reading [Contentful Errors](https://www.contentful.com/developers/docs/references/errors/), they say 

`Errors share the same common attributes as other API resources. They have a sys property, where sys.type is always "Error" and a sys.id that identifies the exact error code (see table below) that occurred. They always have a message property which will be a short description of what went wrong.`

And when I curl the endpoint with an error I get this format back

```
{
  "sys": {
    "type": "Error",
    "id": "NotFound"
  },
  "message": "The resource could not be found.",
  "requestId": "c08a4c02-cb9e-43ea-9c61-79be44a0786c"
}
```

Reply:
Yeah, I'm just wondering if the Node client automatically handles that API response for us and raises an exception, rather than returning it. The documentation above with the `client.getEntry(...).catch(err => console.log(err))` line suggests that it does, but it isn't entirely clear and explicit about it.

Have you tried requesting that endpoint with the Node client rather than with `curl`?

Reply:
So when I do it in with Node I get:

```
await contentful.createClient({accessToken: "sdsdsds", space: "m5k1kmk3zqw"}).getEntries().then(entry => console.log(entry))
Uncaught:
NotFound: {
  "status": 404,
  "statusText": "Not Found",
  "message": "The resource could not be found.",
  "details": {},
  "request": {
    "url": "entries",
    "headers": {
      "Accept": "application/json, text/plain, */*",
      "Content-Type": "application/vnd.contentful.delivery.v1+json",
      "X-Contentful-User-Agent": "sdk contentful.js/9.1.10; platform node.js/v16.14.0; os macOS/v16.14.0;",
      "Authorization": "Bearer ...sdsds",
      "user-agent": "node.js/v16.14.0",
      "Accept-Encoding": "gzip"
    },
    "method": "get"
  },
  "requestId": "f39c6bfd-d02e-4b19-b746-441922e210cf"
}
```

Not clear at all in the error docs. But okay at least we know for certain we need to catch it.